src/ParallelAlgorithms/Interpolation/Targets/ApparentHorizon.hpp

```diff
@@ -136,7 +138,7 @@ namespace TargetPoints {
 ///
 /// For requirements on InterpolationTargetTag, see InterpolationTarget
 template <typename InterpolationTargetTag, typename Frame>
-struct ApparentHorizon {
+struct ApparentHorizon : tt::ConformsTo<intrp::protocols::ComputeTargetPoints> {
```

Comment:
Please add a `static_assert(tt::assert_conforms_to)` to the test for each of the structs you have added protocol conformance to (ApparentHorizon, LineSegment, etc). This will probably fail to compile for the reasons I commented above (DummyTags etc).

Reply:
It will compile, see other comment.

Reply:
Ok good! The `assert_conforms_to` should be part of the tests nonetheless I think. Just a one-line static_assert somewhere.